lib/connection.cpp

```diff
@@ -1430,21 +1430,21 @@ SendToDeviceJob* Connection::sendToDevices(
     const QString& eventType, const UsersToDevicesToContent& contents)
 {
     return callApi<SendToDeviceJob>(BackgroundRequest, eventType,
-                                    generateTxnId(), contents);
+                                    QString::fromUtf8(generateTxnId()), contents);
```

Comment:
```suggestion
                                    QString::fromLatin1(generateTxnId()), contents);
```
And, looking at the hideous back and forth in `sendMessage()`, I tend to think that `generateTxnId()` has to return `QString` in 0.8.